vstools/functions/normalize.py

```diff
@@ -92,7 +94,7 @@ def flatten(items: Any) -> Any:
 def flatten_vnodes(
     *clips: VideoNodeIterable | tuple[VideoNodeIterable, ...], split_planes: bool = False
 ) -> list[vs.VideoNode]:
-    """Flatten a single or multiple video nodes into their planes."""
+    """Flatten a single or multiple video nodes into their individual planes."""
```

Comment:
this is wrong. the previous docstring was wrong too.

this behaves like normal flatten, it just adds a parameter which is split_planes that lets you flatten the nodes even more into their planes